src/julia.h

```diff
@@ -188,7 +188,8 @@ typedef struct _jl_sym_t {
     struct _jl_sym_t *left;
     struct _jl_sym_t *right;
     uptrint_t hash;    // precomputed hash value
-    JL_ATTRIBUTE_ALIGN_PTRSIZE(char name[]);
+    // variable length
+    JL_ATTRIBUTE_ALIGN_PTRSIZE(char name[1]);
```

Comment:
GDB problem can be solved with `const char *(jl_sym_name)(jl_sym_t *sym) {return jl_sym_name(sym);}` ?